dp/cloud/python/magma/db_service/migrations/versions/f09f644ccc67_.py

```diff
@@ -0,0 +1,55 @@
+"""empty message
+
+Revision ID: f09f644ccc67
+Revises: f84efd78a4de
+Create Date: 2022-03-10 14:38:53.050143
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'f09f644ccc67'
+down_revision = 'f84efd78a4de'
```

Comment:
Remember to update it when merging, since this has to point to the latest migration and it might change. Unfortunately we currently don't have any tests for it, so if it is broken, CI won't notice it.